pkg/apis/datavolumecontroller/v1alpha1/types.go

```diff
@@ -46,6 +46,7 @@ type DataVolumeSource struct {
 	S3     *DataVolumeSourceS3     `json:"s3,omitempty"`
 	PVC    *DataVolumeSourcePVC    `json:"pvc,omitempty"`
 	UPLOAD *DataVolumeSourceUpload `json:"upload,omitempty"`
+	CREATE *DataVolumeCreateImage  `json:"create,omitempty"`
```

Comment:
This was something that slipped by me with upload datavolume integration and I think we should probably fix/not perpetuate here.  `HTTP/PVC/S3` are acronyms, so it makes sense for them to be all caps.  `UPLOAD` and `CREATE` are not acronyms so they should be changed to `Upload` and `Create` respectively.

Although, I'm not crazy about the name `Create`.  I wonder if `Empty` of `Blank` would be better?  Or any other suggestions?

Reply:
okay. Will change it to Blank for now.